.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 /scripts/ @Azure/bicep-admins @Azure/avm-core-team-technical-bicep
 /avm/ @Azure/avm-core-team-technical-bicep
 /avm/utilities/ @Azure/avm-core-team-technical-bicep
-#/avm/res/aad/domain-service/ @Azure/avm-res-aad-domainservice-module-owners-bicep
+/avm/res/aad/domain-service/ @Azure/avm-res-aad-domainservice-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Placeholder until team membership is resolved

Reply:
Hi @ReneHezser, please don't resolve the comment until the codeowners file shows no errors.
Relevant azure teams must be created and correct membership must be set up for you to be able to approve future contributions to this module. Ref https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only

Please don't hesitate to reach out if any support needed!

Reply:
Sorry. I'll wait for you to create the necessary groups.

Reply:
Hi @ReneHezser creating contributor and owner groups is part of the module owners' responsabilities. The link above is providing guidelines on how to do that, but in brief you should:
- Create a `avm-res-aad-domainservice-module-owners-bicep` team in the Azure organization
- Add other module owners as team members
- Request the team to be a member of the parent team `avm-technical-reviewers-bicep`
- Wait for someone in the core team to approve the membership
- Create a `avm-res-aad-domainservice-module-contributors-bicep` team in the Azure organization
- Add module contributors as team members
- Request the team to be a member of the parent team `avm-module-contributors-bicep`
- Wait for someone in the core team to approve the membership

Reply:
Thank you @ReneHezser! The membership is now fixed and the codeowners file is valid. Resolving